modules/tensor_mechanics/tests/tensile/tests

```diff
@@ -81,6 +81,46 @@
     abs_zero = 1.0E-5
     cli_args = '--no-trap-fpe'
   [../]
+  [./planar4]
+    type = 'CSVDiff'
+    input = 'planar4.i'
+    csvdiff = 'planar4.csv'
+    abs_zero = 1.0E-8
+    rel_err = 1.0E-5
+    cli_args = '--no-trap-fpe'
```

Comment:
Yes, i get NaN.  It is because PETSc/LAPACK/BLAS explicitly checks 1/0, 0/0, etc, prior to calculating a singular-value decomposition, which i use in the multi-plasticity stuff.   That throws a FPE and moose crashes.